src/index.js

```diff
@@ -134,12 +135,13 @@ const MainAppWithApolloProvider = () => {
     if (globalSettingsPublicJsonFileContent) {
```

Comment:
this needs to be checked more deeply. maybe the change for `setGlobalSettingsState` outside of the `if` was bad.
if an app is installed freshly with 1.2.0, the gobal settings are not working as expected.